generators/form/index.js

```diff
@@ -2,7 +2,6 @@
 const Generator = require('yeoman-generator');
 
 const TEMPLATE_TYPES = {
-  BLANK: 'BLANK',
   SIMPLE: 'SIMPLE',
   COMPLEX: 'COMPLEX',
```

Comment:
Might as well rename these to something like `1-page` and `4-pages`? Because they display in the terminal